src/UserSettingsPage.cpp

```diff
@@ -340,26 +345,57 @@ UserSettingsPage::UserSettingsPage(QSharedPointer<UserSettings> settings, QWidge
 
         formLayout_->addRow(general_);
         formLayout_->addRow(new HorizontalLine{this});
-        boxWrap(tr("Minimize to tray"), trayToggle_);
-        boxWrap(tr("Start in tray"), startInTrayToggle_);
+        boxWrap(
+          tr("Minimize to tray"),
+          trayToggle_,
+          tr("Keep the application running in the background after closing the client window."));
+        boxWrap(tr("Start in tray"),
+                startInTrayToggle_,
+                tr("Start the application in the background without showing the client window."));
         formLayout_->addRow(new HorizontalLine{this});
-        boxWrap(tr("Circular Avatars"), avatarCircles_);
-        boxWrap(tr("Group's sidebar"), groupViewToggle_);
-        boxWrap(tr("Decrypt messages in sidebar"), decryptSidebar_);
-        boxWrap(tr("Show buttons in timeline"), timelineButtonsToggle_);
-        boxWrap(tr("Typing notifications"), typingNotifications_);
-        boxWrap(tr("Sort rooms by unreads"), sortByImportance_);
+        boxWrap(tr("Circular Avatars"),
+                avatarCircles_,
+                tr("Change the appearance of user avatars in chats. OFF - square, ON - Circle."));
+        boxWrap(tr("Group's sidebar"),
+                groupViewToggle_,
+                tr("Show a column containing groups and tags next to the room list."));
+        boxWrap(tr("Decrypt messages in sidebar"),
+                decryptSidebar_,
+                tr("Decrypts the messages shown in the sidebar. Only affects the messages of "
+                   "encrypted chats."));
+        boxWrap(tr("Show buttons in timeline"),
+                timelineButtonsToggle_,
+                tr("Show buttons for reply and additional options next to every message."));
+        boxWrap(tr("Typing notifications"),
+                typingNotifications_,
+                tr("Display a notification containing authors' names whenever someone is writing a "
+                   "message."));
+        boxWrap(tr("Sort rooms by unreads"),
+                sortByImportance_,
+                tr("Display rooms with new messages first."));
```

Comment:
Since this was what started this whole initiative, maybe we should spend a bit more effort on this tooltip?

```suggestion
                tr("Display rooms with new messages first.\nIf this is off the list of rooms will only be sorted by the timestamp of the last message in a room.\nIf this is on rooms which have active notifications (the small circle with a number in it) will be sorted on top. Rooms, that you have muted, will still be sorted by timestamp, since you don't seem to consider them as important as the other rooms."));
```